sources/packages/web/src/types/contracts/Common.ts

```diff
@@ -13,19 +13,20 @@ export interface Address {
   addressLine2?: string;
   city: string;
   postalCode: string;
-  provinceState: string;
+  provinceState?: string;
   country: string;
 }
 
 export interface AddressInfo {
   addressLine1: string;
   addressLine2?: string;
-  province: string;
+  provinceState?: string;
   country: string;
   city: string;
   postalCode: string;
+  selectedCountry?: string;
 }
-export interface ContactInformation {
-  addresses: Array<AddressInfo>;
-  phone: string;
+
+export interface InstitutionLocationInfo {
```

Comment:
Please consider moving this to a more specific file, like institution or location.